conans/client/generators/scons.py

```diff
@@ -0,0 +1,37 @@
+from conans.model import Generator
+
+class SConsGenerator(Generator):
+    @property
+    def filename(self):
+        return "SConscript-conan"
```

Comment:
All the genereators create all lowercase files, and prefer underscore. Is it possible to change the name, or is it somehow dictated by Scons? Maybe something like conanbuildinfo.scons?